src/lightkurve/prf/__init__.py

```diff
@@ -1,2 +1,6 @@
+import warnings
 from .prfmodel import *
-from .tpfmodel import *
+try:
+    from .tpfmodel import *
+except ModuleNotFoundError:
+    warnings.warn("Warning: the tpfmodel module is not available without oktopus")
```

Comment:
Should we add a reference to this issue so users can more easily understand this warning?